tools/file_oi/file_io.py

```diff
@@ -0,0 +1,109 @@
+import xml.etree.ElementTree as ET
+from xml.dom.minidom import parseString
+
+
+def xml_to_string(xml_tag):
+    """
+    Makes xml objects into string formatted in a readable way
+    """
+    return parseString(ET.tostring(xml_tag)).toprettyxml(indent="  ")
+
+
+def parse_config(filename):
+    """
```

Comment:
parameter is not documented

Reply:
Did you mean xml_to_string's parameter wasnt documented?